tests/integration/test_github_workflows.py

```
@@ -107,11 +106,12 @@ class TestWorkflowSyntax:
     @pytest.mark.parametrize("workflow_file", get_workflow_files())
     def test_workflow_valid_yaml_syntax(self, workflow_file: Path):
         """Test that workflow files contain valid YAML syntax."""
-        try:
-            with open(workflow_file, 'r', encoding='utf-8') as f:
-                yaml.safe_load(f)
-        except yaml.YAMLError as e:
-            pytest.fail(f"Invalid YAML syntax in {workflow_file.name}: {e}")
+        if duplicates:
```

Ruff (0.14.7)

Undefined name `duplicates`

(F821)